### ISSUE TITLE
DropDownList accessibility issues

### ISSUE THREAD

**Issue by shaizel** [telerik/kendo-angular#3513]
I'm using the Kendo Angular DropDownList component and I'm facing the same problem, so it's not specific to kendo-react.

I've used Axe DevTools on the first example seen on https://www.telerik.com/kendo-angular-ui/components/dropdowns/dropdownlist/.

I got two issues, the same as those I get in my own application using the DropDownList component.
I've attached the output for both issues below.

First issue and [an explanation of the issue and its solutions](https://dequeuniversity.com/rules/axe/4.3/aria-input-field-name?application=AxeChrome):
![Kendo DropDownList accessibility issues 1](https://user-images.githubusercontent.com/25416352/143089704-86bb1ad8-f3d8-4051-af77-cbca3b402fb2.png)

Second issue and [an explanation of the issue and its solutions](https://dequeuniversity.com/rules/axe/4.3/aria-required-children?application=AxeChrome):
![Kendo DropDownList accessibility issues 2](https://user-images.githubusercontent.com/25416352/143089710-45a792de-946d-425f-a153-7313a73ac33f.png)

**Comment by shaizel** [comment]
There's a workaround for the first issue, ARIA input fields must have an accessible name, found [here](https://github.com/telerik/kendo-angular/issues/2086#issuecomment-649605029).

Using a KendoLabel alongside with the DropDownList places an aria-labelledby attribute on the focusable element of the DropDownList.

The problem is that it forces the use of a KendoLabel, which isn't always needed.

**Comment by yanmariomenev** [comment]
Ref to the FP - 1547812

**Comment by JoomFX** [comment]
With the new version of the Dropdowns, where we made major rendering changes, there are the following accessibility issues with the DropDownList (tested with the Axe DevTools browser extension):

1. ARIA input fields must have an accessible name
2. Certain ARIA roles must contain particular children
3. Buttons must have discernible text

**Issue 1** - When using the KendoLabel together with the DropDownList the label text is added as `aria-labelledby` therefore everything is fine, there is no accessibility issue.
It is correct that there are cases where the KendoLabel is not needed and the DropDownList component needs to be used alone. But in that case we do not have context and do not know what text the screen reader should read.
Therefore it is the user's responsibility to specify the preferred text. And since now, with the new rendering, the focusable element is the host element itself (not an inner element as it used to be), this can be achieved by simply providing a `title` attribute to the DropDownList component like so:

```hml
<kendo-dropdownlist [data]="listItems" title="DropDownList"></kendo-dropdownlist>
```

**Issue 2 and Issue 3** - We will address those in the component and the fix will be available in the next dev package.

**Comment by ipeshev** [comment]
Still reporting 'aria-input-field-name' and 'aria-toggle-field-name' is reported for span toggler, issues 2,3 are not reported. Will keep that "In progress" until solved.

**Comment by JoomFX** [comment]
[This Stackblitz example](https://stackblitz.com/edit/angular-w5q4gi?file=package.json) can be scanned in order for the DropDownList accessibility to be tested.

**NOTES:**
1. If the DropDownList is not used in combination with the Kendo Label, a `title` attribute should be added manually. For more details, please have a look at [the comment above](https://github.com/telerik/kendo-angular/issues/3513#issuecomment-1024297137).
2. A default item must be configured. It could be a placeholder text like `Select item...` as shown in the Stackblitz example.

**Comment by tapopov** [comment]
Fix released in `kendo-angular-dropdowns` v6.0.2
